### ISSUE TITLE
Missing php 8.4 vs17 series on windows.php.net

### ISSUE THREAD

**Issue by dixyes** [winlibs/winlib-builder#30]
Here is missing packages-8.5-vs17-x64-staging.txt: https://windows.php.net/downloads/php-sdk/deps/series/
Will there be a new one or just let me use https://downloads.php.net/~windows/php-sdk/deps/series/packages-8.4-vs17-x64-staging.txt ?
btw, which one is used in PHP official build? Sometimes they are mismatched.

**Comment by cmb69** [comment]
Yes, please use the packages from downloads.php.net; these are used by official builds, and also fetched by the php-sdk-binary-tools 2.3.0 (`phpsdk_deps`).

**Comment by cmb69** [comment]
I'm not sure what to do here. Maybe we should establish the following redirects:

* https://windows.php.net/downloads/php-sdk/deps/ → https://downloads.php.net/~windows/php-sdk/deps/
* https://windows.php.net/downloads/pecl/deps/ → https://downloads.php.net/~windows/pecl/deps/

@shivammathur, @derickr, what do you think?

By the way, since new stable PHP releases have been rolled out, the staging series files should be pushed to stable (PHP 8.1 - 8.4; and possibly PHP 8.5 and master, but these use staging deps by default anyway).

**Comment by shivammathur** [comment]
Fixed in https://github.com/php/infrastructure/pull/6. Now we have redirects from windows.php.net to downloads.php.net
